src/main/java/uk/gov/hmcts/darts/authorisation/component/impl/UserIdentityImpl.java

```diff
@@ -33,38 +34,6 @@ public class UserIdentityImpl implements UserIdentity {
     private final UserAccountRepository userAccountRepository;
     private final UserRolesCourthousesRepository userRolesCourthousesRepository;
 
-    @SuppressWarnings({"PMD.AvoidDeeplyNestedIfStmts", "PMD.CyclomaticComplexity", "PMD.CognitiveComplexity"})
```

Comment:
we should remove the constants EMAILS and PREFERRED_USERNAME from the class if we are not using it here.

Reply:
done